### ISSUE TITLE
GetBayesianHetCoverage hangs indefinitely.

### ISSUE THREAD

**Issue by samuelklee** [broadinstitute/gatk-protected#863]
We've had a few reports of this---let's look into it.

**Comment by mbabadi** [comment]
The only thing that comes to my mind is that GetBayesianHetCoverage enables indexing in samtools/htsjdk by default and GetHetCoverage disables it in most cases. Can you gather a few problematic samples?

**Comment by samuelklee** [comment]
Yup, got a user on the forum that I'm trying to get more information from.  @dlivitz have you run into any samples that you could pass along?

**Comment by LeeTL1220** [comment]
I've had this issue.

**Comment by LeeTL1220** [comment]
This may be related to #865 , since I am having an issue with that as well...

A ton of time is being spent during initialization on ``DiskBasedBAMFileIndex.getSpanOverlapping`` and ``AbstractBamFileIndex.query`` when you have a long interval list.  I am not 100% sure that this is the same issue as here, but symptoms look the same.  I saw 50k intervals taking about 50 sec to init, but when I ran 500k intervals I just killed it after half an hour.
